client/src/components/application/Examine/RequestInfoHeader.vue

```diff
@@ -267,6 +271,14 @@ export default {
         },
       }
 
+      if (this.compName3) {
```

Comment:
Looks like this validation fails when no name2 and 3 exist at all. So that's a bug - we'll never pass edit validation if there is only one name.